src/main/java/skytils/skytilsmod/features/impl/misc/ItemFeatures.java

```diff
@@ -93,7 +93,10 @@ public void onSlotClick(GuiContainerEvent.SlotClickEvent event) {
                             event.setCanceled(true);
                             return;
                         }
+                        if (!extraAttr.hasKey("baseStatBoostPercentage")) {
```

Comment:
This code won't run if protect starred items is not enabled, is this intended?

Reply:
Yea, was thinking if I should make it work with highlight, protect starred items or make a new feature, but i feel like its too minor for a new feature and doesnt make sense with highlight